addons/binding/org.openhab.binding.mqtt.generic/README.md

```diff
@@ -258,16 +260,30 @@ Bridge mqtt:broker:myUnsecureBroker [ host="192.168.0.42", secure=false ]
 }
 ```
 
-demo.items:
+demo.Things (using non-bridge setup):
```

Comment:
But the wording is strange. It is not a "non-bridge" setup. The bridge is just separately declared. This is true for all OH 2 bindings. I really don't understand why we need this to the detail in the MQTT documentation. Do people not read the official documentation?

Reply:
I like to keep parts of this PR. But could you please remove what is written in parentheses.